build_tools/circle/build_doc.sh

```diff
@@ -123,6 +123,7 @@ conda create -n $CONDA_ENV_NAME --yes --quiet python="${PYTHON_VERSION:-*}" \
 source activate testenv
 pip install sphinx-gallery
 pip install sphinxext-opengraph
```

Comment:
Indeed, clearly the setup.cfg extra-requires doc section is not used.

Maybe we should modify the present script to use it?

Reply:
Something like
```
pip install -e .[doc]
```
should work, and could allow replacing the list above